From: Jakub Onderka <ahoj@jakubonderka.cz>
Date: Mon, 27 Apr 2020 16:08:52 +0200
Subject: [PATCH] Fix argument order for implode method

Different order is deprecated and will cause fatal error in PHP8
Origin: upstream, https://github.com/cakephp/cakephp/commit/fc3b50c523372691a26cd0f97cd6f60ec11fa9a4;
---
 system/xi3/lib/Cake/Console/ConsoleOutput.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/system/xi3/lib/Cake/Console/ConsoleOutput.php b/system/xi3/lib/Cake/Console/ConsoleOutput.php
index 4690a85..1f52df0 100644
--- a/system/xi3/lib/Cake/Console/ConsoleOutput.php
+++ b/system/xi3/lib/Cake/Console/ConsoleOutput.php
@@ -265,7 +265,7 @@ class ConsoleOutput {
 				$styleInfo[] = static::$_options[$option];
 			}
 		}
-		return "\033[" . implode($styleInfo, ';') . 'm' . $matches['text'] . "\033[0m";
+		return "\033[" . implode(';', $styleInfo) . 'm' . $matches['text'] . "\033[0m";
 	}
 
 /**
